Makefile

```diff
@@ -12,21 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Use the native vendor/ dependency system
+export GO15VENDOREXPERIMENT=1
+
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 BUILD_DIR ?= ./out
+REPOPATH ?= k8s.io/minikube
+BUILD_IMAGE ?= gcr.io/google_containers/kube-cross:v1.6.2-1
 
 ifeq ($(IN_DOCKER),1)
 	GOPATH := /go
 else
 	GOPATH := $(shell pwd)/_gopath
 endif
 
-REPOPATH ?= k8s.io/minikube
-export GO15VENDOREXPERIMENT=1
-
-# Set the version information in kubernetes.
-LD_FLAGS = "-s -w $(shell python hack/get_k8s_version.py)"
+# Set the version information for the Kubernetes servers, and build localkube statically
+LD_FLAGS = "-s -extldflags '-static' -w $(shell docker run --rm -it -v $(shell pwd):/minikube -w /minikube python python hack/get_k8s_version.py)"
```

Comment:
nit: this gets a bit hard to read, think you could break out the get_k8s_version part to a separate variable?